### PR TITLE
Adapt to new build symbols (requires 2.434)

### PR DESCRIPTION
**Pull request by timja**
Fixes https://github.com/jenkinsci/dark-theme-plugin/issues/399

### Testing done

Created a few matrix projects.

Tested nobuild, aborted, success.
Icons all look good (better than before as there was some rendering issues previously with text too close to icons)

image

image

image

image

image

```[tasklist]
### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

**Review by NotMyFault — APPROVED** [review]
Tested locally, looks nice so far

**Comment by Pesa** [comment]
ping?

File: src/main/resources/hudson/matrix/MatrixBuild/ajaxMatrix.jelly [hunk 1]
```diff
@@ -32,11 +32,12 @@ THE SOFTWARE.
       <j:set var="b" value="${p.run}" />
       <j:choose>
         <j:when test="${b==null}">
-          <l:icon class="icon-nobuilt icon-md" tooltip="${%Not run}" alt="${%Not run}"/>
+          <l:icon src="symbol-status-nobuilt" class="icon-md" tooltip="${%Not run}" alt="${%Not run}"/>
+          ${%Not configured}
```
Comment (Pesa) [inline src/main/resources/hudson/matrix/MatrixBuild/ajaxMatrix.jelly:36]: should this be "Not run" to match `tooltip` and `alt`?

Comment (timja) [reply src/main/resources/hudson/matrix/MatrixBuild/ajaxMatrix.jelly:36]:
```suggestion
          ${%Not run}
```

**Comment by timja** [comment]
@batmat @rsandell @amuniz @MRamonLeon ?

**Review by rsandell — APPROVED** [review]

**Review by Pesa — APPROVED** [review]

**Comment by NotMyFault** [comment]
Anything we are waiting for in particular? 👀

**Comment by timja** [comment]
Nothing I’m aware of. I updated it to LTS release in case that’s what held back a merge before

**Comment by Pesa** [comment]
bump

File: pom.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <properties>
       <changelist>999999-SNAPSHOT</changelist>
       <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-      <jenkins.version>2.401.3</jenkins.version>
+      <jenkins.version>2.440.1</jenkins.version>
```
Comment (basil) [inline pom.xml:18]:
```suggestion
      <jenkins.version>2.440.3</jenkins.version>
```

**Comment by lupalby** [comment]
Not entirely sure, but there is a chance that this PR or something related to this is breaking icons and clickable links in the matrix after the update to the latest Jenkins LTS. Others and I are affected: https://issues.jenkins.io/browse/JENKINS-73225

**Pull request merged**